asciidoc.py

```diff
@@ -1296,9 +1264,13 @@ class Lex:
     prev_element = None
     prev_cursor = None
     def __init__(self):
-        raise AssertionError,'no class instances allowed'
+        raise AssertionError('no class instances allowed')
+
+    def __iter__(self):
+        return self
+
     @staticmethod
-    def next():
+    def __next__():
```

Comment:
I think this is better named `next_element()` not `__next__()` since it is not the iterator next function.